doc/sphinx/language/core/records.rst

```diff
@@ -78,36 +150,44 @@ in which case the correctness of :n:`@type__3` may rely on the instance :n:`@ter
    Note here that the fields ``Rat_bottom_cond`` depends on the field ``bottom``
    and ``Rat_irred_cond`` depends on both ``top`` and ``bottom``.
 
-Let us now see the work done by the :cmd:`Record` command. First the command
-generates a variant type definition with just one constructor:
-:n:`Variant @ident {* @binder } : @sort := @ident__0 {* @binder }`.
-
-To build an object of type :token:`ident`, provide the constructor
-:n:`@ident__0` with the appropriate number of terms filling the fields of the record.
-
 .. example::
 
-   Let us define the rational :math:`1/2`:
+   Let us define the rational :math:`1/2` using either the applicative or record syntax:
```

Comment:
Note that Coq comments in examples will appear in the manual--probably better to put "(\* applicative form \*)" and "(* record form *)" into the body of the example for readability.

Reply:
The doc uses Python code to extract individual sentences and comments.  It's crude.  I think comments only work between sentences, not within them.